### PR TITLE
Rename events to past-tense

### PR DESCRIPTION
**Pull request by elopio**
# 🚀 Description

Following our renaming discussion, all event names should be in past tense. This renames all the events that were not using this rule, except the ones that are defined in ERCs.

- [x] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](../blob/master/CONTRIBUTING.md)
- [x] ✅ I've added tests where applicable to test my new functionality.
- [x] 📖 I've made sure that my contracts are well-documented.
- [x] 🎨 I've run the JS/Solidity linters and fixed any issues (`npm run lint:all:fix`).

### CODE REVIEW

**Comment by elopio** [comment]
I have a remaining question: is `Withdrawn` properly named?

**Comment by nventuro** [comment]
Depends, are we using simple past or past participle? In that case, it should be named `Withdrew`. @shrugs?

**Comment by shrugs** [comment]
uh, my gut says `Withdrawn` is correct. Withdrew would make sense if the topic of the event was a user like `UserWithdrew(address _user, uint256 _amount)` but it still doesn't feel better than `AmountWithdrawn(address _user, uint256 _amount)`

File: CODE_STYLE.md [hunk 1]
```diff
@@ -23,3 +23,6 @@ function test(uint256 _testParameter1, uint256 _testParameter2) {
     ...
 }
 ```
+
+* Events should be emitted immediately after the state change that they
+  represent, and consequently they should be named in past tense.
```
Comment (nventuro) [inline CODE_STYLE.md]: Should we provide a couple examples? I'd add a note mentioning that some standards (e.g. ERC20) use present tense, and in those cases the spec prevails.

Comment (elopio) [reply CODE_STYLE.md]: Good ideas @nventuro! Please check if you like it.

Comment (nventuro) [reply CODE_STYLE.md]: Great!

**Comment by frangio** [comment]
It's `Withdrawn` because it's shorthand for "was withdrawn" which is in the simple past.

**Pull request merged**